product-mini/platforms/posix/main.c

```diff
@@ -294,6 +294,7 @@ static char global_heap_buf[10 * 1024 * 1024] = { 0 };
 int
 main(int argc, char *argv[])
 {
+    int32 ret = -1;
```

Comment:
The compiler complains "warning: variable ‘ret’ set but not used" when `-DWAMR_BUILD_SPEC_TEST=1`. How about:
```
#if WASM_ENABLE_SPEC_TEST == 0
    int32 ret = -1;
#endif